app/models/user.rb

```diff
@@ -54,6 +55,19 @@ def belonging_to?(room)
     own?(room) || participating?(room)
   end
 
+  def evaluated?(room)
+    evaluations.find_by(room: room).present?
+  end
+
+  def evaluations_score(output = nil, round = nil)
+    length = owned_rooms.sum {|room| room.evaluations.length }
+    return output if length.zero?
+
+    total = owned_rooms.sum {|room| room.evaluations_score }
```

Comment:
__[RuboCop]__

Style/SymbolProc: Pass `&:evaluations_score` as an argument to `sum` instead of a block.


<!-- Comment by [Sider](https://sider.review) -->